ui/admin/app/components/form/target/details/index.js

```diff
@@ -2,16 +2,23 @@ import Component from '@glimmer/component';
 import { types } from 'api/models/target';
 
 //Note: this is a temporary solution till we have resource type helper in place
-const icons = ['network', 'terminal'];
+const icons = {
+  ssh: 'terminal-screen',
+  tcp: 'network',
+};
 
 export default class FormTargetComponent extends Component {
   // =properties
   /**
    * maps resource type with icon
    * @type {object}
    */
-  get mapResourceTypeWithIcon() {
-    return types.reduce((obj, type, i) => ({ ...obj, [type]: icons[i] }), {});
+  get typeMetas() {
+    const reversedTypes = [...types].reverse();
```

Comment:
Any particular reason for reversing this? Can we do 
```
const icons = {
  tcp: 'network',
  ssh: 'terminal-screen',
};
```

Reply:
Because TCP is listed first in the model types (which makes sense), but on display we want SSH first.  The order of keys in an object is not necessarily the same as insertion order or code order, so going forward an array is a better option.

This is temporary anyway and will eventually be replaced by a helper.

Reply:
Ahh! that is correct regarding the order of keys in an object. 🙈